miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -19,7 +19,7 @@ internal class MiniAppWebChromeClient(
     @Suppress("TooGenericExceptionCaught", "SwallowedException")
     @VisibleForTesting
     internal val bridgeJs = try {
-        val inputStream = context.assets.open("bridge.js")
+        val inputStream = context.assets.open("js-miniapp/js-miniapp-bridge/export/android/bridge.js")
```

Comment:
As the script is exported from js-miniapp repo. In Android repo, it could use `assets/imports` type of path. What do you think?

Reply:
Do you mean it will become `imports/js-miniapp/js-miniapp-bridge/export/android/bridge.js`?

Reply:
@minh-rakuten Probably, `assets/imports/mini-app-bridge/bridge.js`.

Reply:
The path `js-miniapp/js-miniapp-bridge/export/android/bridge.js` is fixed from js repo. I don't know how to change that default path.

Reply:
With sparse checkouts in place, it will require changing the structure in js-miniapp repo firstly and discarding the export-import structure. So the path could simply look like: `js-miniapp/comm-bridge/platform/android/bridge.js`